src/planning.py

```diff
@@ -225,6 +231,7 @@ def _run_low_level_search(
         option = nsrt.sample_option(state, rng_sampler)
         plan[cur_idx] = option
         if option.initiable(state):
+            print(f"Trying option {option}")
```

Comment:
to keep the printouts readable in the other environments, let's remove or comment out this new print and the ones below